### PR TITLE
update chinese translation of project_burnout.md

### PR DESCRIPTION
**Pull request by ME-Msc**

### CODE REVIEW

**Comment by eyehwan** [comment]
@ME-Msc  Hi, please execute git commit -sm "" to sign off dco. :)

File: chinese/wg-diversity-inclusion/focus-areas/project-and-community/project-burnout.md [hunk 1]
```diff
@@ -4,64 +4,68 @@
...
+  - 如何判断项目贡献者的幸福感？
 
 ## 参考资料
 
 1. [关于开源社区中的倦怠你需要知道什么](https://opensource.com/article/19/11/burnout-open-source-communities)
-2. [一个开源维护者在精疲力竭之后会做什么？](https://www.infoworld.com/article/3563326/what-does-an-open-source-maintainer-do-after-burnout.html)
-3. [Raman, N., Cao, M., Tsvetkov, Y., Kästner, C., & Vasilescu, B. (2020).开放源码中的压力和倦怠: 走向发现、理解和减轻不健康的互动。在国际软件工程会议上，新思想和新成果(ICSE-NIER)](https://cmustrudel.github.io/papers/raman20toxicity.pdf)
-4. [2020年领导力报告中的情绪适应力](https://docs.google.com/document/d/18FfZ86PGA_uSFf425EzKXAmiFQLFBPqjqPN7iu1TZRw/edit#) (_“但不是针对开源软件”_)
-5. 帮助人们确定他们处于哪个倦怠阶段，或者他们是否处于一个更严重的倦怠阶段: [避免过度疲劳和过上幸福生活的实用指南](https://opensource.com/business/15/12/avoid-burnout-live-happy)
+2. [一个开源维护者在倦怠之后会做什么？](https://www.infoworld.com/article/3563326/what-does-an-open-source-maintainer-do-after-burnout.html)
```
Comment (eyehwan) [inline chinese/wg-diversity-inclusion/focus-areas/project-and-community/project-burnout.md:60]: "倦怠" is much better than "精疲力竭", thanks.

**Review by eyehwan — APPROVED** [review]
Looks great!

File: chinese/wg-diversity-inclusion/focus-areas/project-and-community/project-burnout.md (hunk 1 above)
Comment (ME-Msc) [reply chinese/wg-diversity-inclusion/focus-areas/project-and-community/project-burnout.md:60]: My pleasure.

**Pull request merged**